packages/kbn-cell-actions/src/components/hover_actions_popover.tsx

```diff
@@ -40,23 +40,30 @@ interface Props {
   visibleCellActions: number;
   actionContext: CellActionExecutionContext;
   showActionTooltips: boolean;
+  disabledActions: string[];
```

Comment:
```suggestion
  disabledActions?: string[];
```

Reply:
This is intentional. `CellActions` receives optional props and initializes them with default values; That way, private components properties are compulsory.